modules/swagger-codegen/src/main/resources/swift3/Configuration.mustache

```diff
@@ -0,0 +1,14 @@
+// Configuration.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
+
+open class Configuration {
+	
+	// This value is read only once. 
+    open static var dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
```

Comment:
> This value is used to configure the date formatter that is used to serialize dates into JSON format. You must set it prior to encoding any dates, and it will only be read once.